addons/purchase/models/purchase_order_line.py

```diff
@@ -31,6 +31,7 @@ class PurchaseOrderLine(models.Model):
         digits='Discount',
         store=True, readonly=False)
     taxes_id = fields.Many2many('account.tax', string='Taxes', domain=['|', ('active', '=', False), ('active', '=', True)], context={'active_test': False})
+    tax_id = fields.Many2many(related='taxes_id', string='Taxes (alias)')
```

Comment:
```suggestion
    tax_ids = fields.Many2many(related='taxes_id', string='Taxes (alias)')
```

Reply:
Cosmetic changes are not in the scope of this PR (and will explode the size of the diff).